packages/react-components/src/components/VideoGallery/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '../DevicesButton';
+
+/**
+ * @internal
+ */
+export interface LocalVideoCameraCycleButtonProps {
+  /** Array of cameras available to the user. */
+  cameras: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  selectedCamera: OptionsDevice;
+  /** callback function to change video feed. */
+  onSelectCamera: (device: OptionsDevice) => Promise<void>;
+}
+
+const localVideoCameraCycleButtonStyles: IButtonStyles = {
+  root: {
+    position: 'absolute',
+    right: '0.2rem',
+    top: '0.2rem',
+    color: '#FFFFFF', // only shows up on running video feed to we want to force specific colours.
+    zIndex: 5, // to layer the button above the video tile.
```

Comment:
Please add constants for all relevant zIndex together. (i.e., if this is 5, what is 4?)